### PR TITLE
Added GenerateFromKeys method for client and RequestGenerateFromKeys, ResponseGenerateFromKeys structs

### PR DESCRIPTION
**Pull request by ai-da-stas**

### CODE REVIEW

File: wallet/structs.go [hunk 1]
```diff
@@ -1017,6 +1017,32 @@ type RequestCreateWallet struct {
...
+	Address string `json:"address"`
+	// (Optional - omit to create a view-only wallet) The wallet's private spend key.
+	SpendKey string `json:"spendkey"`
+	// The wallet's private view key.
+	ViewKey string `json:"viewkey"`
+	// The wallet's password.
+	Password string `json:"password"`
+	// (Optional) If true, save the current wallet before generating the new wallet. (Defaults to true)
+	AutoSaveCurrent bool `json:"autosave_current"`
+}
```
Comment (rbrunner7) [inline wallet/structs.go:1038]: About 2 years ago this struct got an additional member to set the seed language of the new wallet:

    std::string language;

Not sure you leave out that on purpose or you worked with some documentation that does not yet contain it.

**Review by rbrunner7 — APPROVED** [review]
Looks good to me, except that seed language struct member that I commented on.

My go knowledge is limited, but anyway, by comparing the new call with existing ones, and using my general knowledge about Monero RPC, I am quite sure the code is ok.

File: wallet/structs.go (hunk 1 above)
Comment (ai-da-stas) [reply wallet/structs.go:1038]: Thank you very much! Indeed, this parameter is not yet specified in the wallet-rpc documentation. I have made the necessary corrections.

**Comment by omani** [comment]
@lepehovsv pls remember to delete your comment on the go-tdlib project.

**Comment by ai-da-stas** [comment]
> @lepehovsv pls remember to delete your comment on the go-tdlib project.

@omani Everything is done, thank you very much!

**Pull request merged**